models/nft/ethereum/nft_ethereum_top_minters.sql

```diff
@@ -0,0 +1,78 @@
+{{ config(
+    alias = 'top_minters',
+    materialized='incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['minter', 'nft_contract_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "sector",
+                                "nft",
+                                \'["Henrystats"]\') }}'
+    )
+}}
+
+WITH 
+
+src_data as (
+    {% if not is_incremental() %}
+    SELECT 
+        nft_contract_address,
+        buyer as minter, 
+        SUM(amount_original) as eth_spent, 
+        COUNT(*) as no_minted,
+        NOW() - Interval '5 Minutes' as last_updated
+    FROM 
+    {{ ref('nft_mints') }}
+    WHERE blockchain = 'ethereum'
+    AND currency_symbol IN ('WETH', 'ETH')
+    AND amount_original IS NOT NULL
+    AND block_time < NOW() - Interval '5 Minutes'
```

Comment:
we should use the same materialisation interval as other spells, at the moment we don't expect to have less than 1 hour freshness. Sorry if this got lost in translation! 

I'd suggest using:
- `max(block_time)` as the value for `last_updated`
- `block_time > (select max(last_updated) from {{this}} )` as the incremental filter

I think we can reasonably tolerate discrepancies caused not using the usual 7 day window here, but if someone disagrees please flag.

Reply:
looks good to me, I'll update!

Reply:
fyi, using the `block_time > (select max(last_updated) from {{this}} )` type logic has caused us issues in the past performance wise, which is why we switched to hardcoded seven day period. i'd love to have that revisited in the future, especially as duneSQL is more prevalant and may allow for this logic

Reply:
👀🥲